lib/carrierwave/uploader/download.rb

```diff
@@ -43,7 +43,8 @@ def file
           @file
 
         rescue StandardError => e
-          raise CarrierWave::DownloadError, "could not download file: #{e.message}"
+          reason = e.is_a?(Net::HTTPBadResponse) || e.is_a?(Errno::EHOSTUNREACH) ? '' : ": #{e.message}"
```

Comment:
Can you explain this further? Why are messages swallowed for these two exception classes?